branchtesting/src/main/java/branchexample/App.java

```diff
@@ -10,7 +10,8 @@ public void aMethod( String[] args )
     		// TODO fix this too..
     		int j = 5 / i;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "j". ([squid:S1854](http://localhost:9721/coding_rules#rule_key=squid%3AS1854))

[See it in SonarQube](http://localhost:9721/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=2&issues=AWU6DJQJcMk10FtshGW-&open=AWU6DJQJcMk10FtshGW-)

[ISSUE_KEY]: AWU6DJQJcMk10FtshGW-